core/src/main/java/org/elasticsearch/action/suggest/SuggestRequestBuilder.java

```diff
@@ -45,7 +45,7 @@ public SuggestRequestBuilder(ElasticsearchClient client, SuggestAction action) {
     /**
      * Add a definition for suggestions to the request
      */
-    public <T> SuggestRequestBuilder addSuggestion(SuggestionBuilder<T> suggestion) {
+    public <T extends SuggestionBuilder> SuggestRequestBuilder addSuggestion(SuggestionBuilder<T> suggestion) {
```

Comment:
I think we can use the unbounded wildcard here since SuggestBuilder will just add to `List<SuggestionBuilder<?>> suggestions` anyway.